daemon/mgr/container_storage.go

```diff
@@ -169,6 +169,8 @@ func (mgr *ContainerManager) getMountPointFromBinds(ctx context.Context, c *Cont
 			return err
 		}
 
+		mp.CopyData = false
```

Comment:
I think the right way is like below：
```
} else {
    mp.CopyData = false
}
```

As your method, it will overwrite the default `CopyData=true`

Reply:
You are right. A new version will fix that